packages/terra-worklist-data-grid/src/subcomponents/Row.module.scss

```diff
@@ -3,12 +3,18 @@
 @import '../clinical-lowlight-theme/Row.module';
 
 :local {
+  tr.selectable:hover {
```

Comment:
I would use .worklsit-data-grid-row instead of the tr tag name.  Using the class name would be a better option.

Reply:
Updated